source/deploy/mattermost-changelog.md

```diff
@@ -11,6 +11,108 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
+```
+
+### Improvements
+
+#### User Interface (UI)
+ - Pre-packaged Calls plugin version [v0.27.0](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.27.0).
+ - Pre-packaged Jira plugin version [v4.1.1](https://github.com/mattermost/mattermost-plugin-jira/releases/tag/v4.1.1).
+ - Pre-packaged GitLab plugin version [v1.9.0](https://github.com/mattermost/mattermost-plugin-gitlab/releases/tag/v1.9.0).
+ - Pre-packaged Zoom plugin version [v1.8.0](https://github.com/mattermost/mattermost-plugin-zoom/releases/tag/v1.8.0).
+ - Updated the default themes to reduce eye strain (particularly on the dark themes).
+ - Added UI improvements to the core layout.
+ - Improved error text when inviting guests to a team.
+ - Increased the visibility Mattermost edition in-product when using the free edition.
+ - Added an **Unsupported** label to the Team/E0 editions in the product menu.
+ - Improved the look and feel of the **True/False** selector in the System Console.
+ - Updated the channel header layout to reduce height and simplify the UI.
+
+#### Administration
+ - Removed safety limit error message in compiled Team Edition and unlicensed Enterprise Edition deployments when message count exceeds 5 million posts.
+ - Adjusted safety limit error message to show when users exceed 5,000 in compiled Team Edition and Enterprise Edition deployments when enterprise scale and access control automation features are unavailable. ``ERROR_SAFE_LIMITS_EXCEEDED``.
+ - Improved the message length validation step in the ``mmctl import validate`` command.
+ - Added shell completion to ``mmctl user active`` and ``mmctl user deactivate``.
+ - Removed support for self-serve purchases of Mattermost Subscriptions in various flows, throughout Cloud and Self Hosted environments.
+ - Removed support for self-serve true up review submission in the **System Console**. 
+ - Added streaming support to the file attachments import process.
+ - Added LDAP job command to mmctl.
+ - Made LDAP sync jobs more resilient against errors.
+ - Removed the ``PostPriority`` feature flag.
+ - Improved the error message of ``NotFound`` errors in store.
+ - Added support for post priority to incoming webhooks and outgoing webhook responses.
+ - Added a validation that the payload for an open Interactive Dialog request is valid according to the rules at https://developers.mattermost.com/integrate/plugins/interactive-dialogs/.
+ - Unblocked notification calls by using usernames instead of full names in case of a missing user profile.
+ - Increased the maximum password limit from 64 to 72 characters (``PasswordMaximumLength``).
+
+#### Performance
+ - Added the initial version of new client-side performance metrics to track web app performance and can be monitored in new [Grafana board](https://grafana.com/grafana/dashboards/21305).
```

Comment:
@amyblais this will be a different Grafana board than the metrics one. We are still waiting on the link for this one. @hmhealey is working on uploading this to Grafana.com

Reply:
@marianunez I'm merging this changelog to the release branch in prep for tomorrow's release day https://github.com/mattermost/docs/pull/7137. Ping me when the dashboard link is ready.